docs/dev/test-architecture.adoc

```diff
@@ -238,10 +244,16 @@ Expect(output).To(ContainSubstring("No file changes detected, skipping build"))
 ----
 +
 
-* If oc, odo or generic library you are looking for is not present in helper package then create a new library function as per the scenario requirement.
+* If oc, odo or generic library you are looking for is not present in helper package then create a new library function as per the scenario requirement. 
```

Comment:
I think this can actually be removed. Sort of self explanatory to create a new library function if it's not available...

Reply:
Instead of just removing this I would kind of more inclined towards combining https://github.com/openshift/odo/pull/3726/files#diff-27813ce26d5cb87e1c303086adb6db24R247-R250